### PR TITLE
Update Beyond Compare.app to 4.0.3.19420 and add zap stanza

### PR DESCRIPTION
**Pull request by sanchezm**

### CODE REVIEW

File: Casks/beyond-compare.rb [hunk 1]
```diff
@@ -11,4 +11,11 @@
   postflight do
     suppress_move_to_applications
   end
+
+  zap :delete => [
+                  '~/Library/Application Support/Beyond Compare',
+                  '~/Library//Caches/com.apple.helpd/Generated/com.ScooterSoftware.BeyondCompare.help',
```
Comment (vitorgalvao) [inline Casks/beyond-compare.rb]: There’s an extra `/ ` after `Library` (on the next two lines as well).

**Comment by sanchezm** [comment]
sorry about that, got some bogus /'s in my editor somehow.  Thanks for the catch.

**Comment by vitorgalvao** [comment]
Looks good, now. Thank you. Only thing left is [squashing](http://davidwalsh.name/squash-commits-git)?

**Comment by sanchezm** [comment]
sorry forgot to squash ... should be good now.

**Comment by ndr-qef** [comment]
Thanks. Merged as 86de52b, with the following changes:
- cleaned up commit message;
- trimmed trailing whitespace.

**Pull request closed without merging**